.agents/skills/qodo-get-rules/references/search-endpoint.md

```
@@ -0,0 +1,157 @@
+
+curl -s -X POST \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer ${API_KEY}" \
+  -H "request-id: ${REQUEST_ID}" \
+  -H "qodo-client-type: skill-qodo-get-rules" \
+  -d "${BODY}" \
+  "${API_URL}/rules/search"
+```
+
+With optional trace header:
+```bash
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)